fabric/message_bus/consumer.py

```diff
@@ -0,0 +1,440 @@
+#!/usr/bin/env python3
+# #######################################################################
+# Copyright (c) 2020 RENCI. All rights reserved
```

Comment:
should change to Fabric Testbed MIT Licence?

Reply:
License updated!